### PR TITLE
impl(delayedack): Added log context to delayedack-middleware

### PR DESCRIPTION
**Pull request by omritoptix**
## Description

----

Closes #XXX

**All** items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow-up issues.

PR review checkboxes:

I have...

- [ ]  Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ]  Targeted PR against the correct branch
- [ ]  included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ]  Linked to the GitHub issue with discussion and accepted design
- [ ]  Targets only one GitHub issue
- [ ]  Wrote unit and integration tests
- [ ]  Wrote relevant migration scripts if necessary
- [ ]  All CI checks have passed
- [ ]  Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code)
- [ ]  Updated the scripts for local run, e.g genesis_config_commands.sh if the PR changes parameters
- [ ]  Add an issue in the [e2e-tests repo](https://github.com/dymensionxyz/e2e-tests) if necessary

SDK Checklist
- [ ] Import/Export Genesis
- [ ] Registered Invariants
- [ ] Registered Events
- [ ] Updated openapi.yaml
- [ ] No usage of go `map`
- [ ] No usage of `time.Now()`
- [ ] Used fixed point arithmetic and not float arithmetic
- [ ] Avoid panicking in Begin/End block as much as possible
- [ ] No unexpected math Overflow
- [ ] Used `sendCoin` and not `SendCoins`
- [ ] Out-of-block compute is bounded
- [ ] No serialized ID at the end of store keys
- [ ] UInt to byte conversion should use BigEndian

Full security checklist [here](https://www.faulttolerant.xyz/2024-01-16-cosmos-security-1/)

----;

For Reviewer:

- [ ]  Confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ]  Reviewers assigned
- [ ]  Confirmed all author checklist items have been addressed

---;

After reviewer approval:

- [ ]  In case the PR targets the main branch, PR should not be squash merge in order to keep meaningful git history.
- [ ]  In case the PR targets a release branch, PR must be rebased.

### CODE REVIEW

File: x/delayedack/ibc_middleware.go [hunk 1]
```diff
@@ -46,7 +46,12 @@ func (im IBCMiddleware) OnRecvPacket(
 	if !im.keeper.IsRollappsEnabled(ctx) {
 		return im.IBCModule.OnRecvPacket(ctx, packet, relayer)
 	}
-	logger := ctx.Logger().With("module", "DelayedAckMiddleware")
+	logger := ctx.Logger().With(
```
Comment (mtsitrin) [inline x/delayedack/ibc_middleware.go:49]: it has some duplicates with the 
`logger.Debug("Saving rollapp packet"`
that follows

File: x/delayedack/ibc_middleware.go [hunk 2]
```diff
@@ -46,7 +46,11 @@ func (im IBCMiddleware) OnRecvPacket(
...
+		"module", "DelayedAckMiddleware",
```
Comment (danwt) [inline x/delayedack/ibc_middleware.go:50]: could use moduleName const?

File: x/delayedack/ibc_middleware.go [hunk 3]
```diff
@@ -95,8 +99,10 @@ func (im IBCMiddleware) OnRecvPacket(
 	}
 	im.keeper.SetRollappPacket(ctx, rollappPacket)
 
-	logger.Debug("Saving rollapp packet", "rollappID", rollappPacket.RollappId, "src channel", rollappPacket.Packet.SourceChannel,
-		"sequence", rollappPacket.Packet.Sequence, "proofHeight", rollappPacket.ProofHeight, "type", rollappPacket.Type)
+	logger.Debug("Saving rollapp packet",
```
Comment (danwt) [inline x/delayedack/ibc_middleware.go:102]:
```suggestion
	logger.Debug("Set rollapp packet",
```

File: x/delayedack/ibc_middleware.go [hunk 4]
```diff
@@ -116,7 +122,11 @@ func (im IBCMiddleware) OnAcknowledgementPacket(
 	if !im.keeper.IsRollappsEnabled(ctx) {
 		return im.IBCModule.OnAcknowledgementPacket(ctx, packet, acknowledgement, relayer)
 	}
-	logger := ctx.Logger().With("module", "DelayedAckMiddleware")
+	logger := ctx.Logger().With(
+		"module", "DelayedAckMiddleware",
```
Comment (danwt) [inline x/delayedack/ibc_middleware.go:126]: ditto modulename

**Review by danwt — APPROVED** [review]
approved but left some suggestions

File: x/delayedack/ibc_middleware.go (hunk 2 above)
Comment (zale144) [inline x/delayedack/ibc_middleware.go:50]: I'm just nitpicking, the actual module name is `delayedack`

**Review by zale144 — APPROVED** [review]
lgtm

**Pull request merged**